problems/my_inject.rb

```diff
@@ -1,5 +1,5 @@
 class Array
-  # Monkey patch the Array class and add a my_inject method. If my_inject receives
+  # Write an `Array#my_inject` method. If my_inject receives
```

Comment:
Add note similar to above explicitly stating that they are not allowed to use built-in `Array#inject` or `Array#reduce` methods.